### ISSUE TITLE
Fingerbank will not install after a purge

### ISSUE THREAD

**Issue by dwlfrth** [inverse-inc/fingerbank#25]
If the previous fingerbank package has been purged (tested on Debian), the fingerbank group is left behind because the 'pf' user is a member: 

```
$ sudo dpkg --purge --force-all fingerbank
dpkg: fingerbank: dependency problems, but removing anyway as you requested:
 packetfence depends on fingerbank (>= 1.0.4); however:
  Package fingerbank is to be removed.

(Reading database ... 69449 files and directories currently installed.)
Removing fingerbank ...
userdel: group fingerbank not removed because it has other members.
Purging configuration files for fingerbank ...
```

Further installation will then fail:

```
$ sudo apt-get install fingerbank 
Reading package lists... Done
Building dependency tree       
Reading state information... Done
The following NEW packages will be installed:
  fingerbank
0 upgraded, 1 newly installed, 0 to remove and 0 not upgraded.
Need to get 20.2 MB of archives.
After this operation, 135 MB of additional disk space will be used.
Get:1 http://inverse.ca/downloads/PacketFence/debian/ wheezy/wheezy fingerbank all 1.0.4-1 [20.2 MB]
Fetched 20.2 MB in 15s (1,280 kB/s)                                                                                  
Selecting previously unselected package fingerbank.
(Reading database ... 69366 files and directories currently installed.)
Unpacking fingerbank (from .../fingerbank_1.0.4-1_all.deb) ...
useradd: group fingerbank exists - if you want to add this user to that group, use -g.
dpkg: error processing /var/cache/apt/archives/fingerbank_1.0.4-1_all.deb (--unpack):
 subprocess new pre-installation script returned error exit status 9
Errors were encountered while processing:
 /var/cache/apt/archives/fingerbank_1.0.4-1_all.deb
E: Sub-process /usr/bin/dpkg returned an error code (1)
```

**Comment by dwlfrth** [comment]
Should be done the same way we are doing it with RHEL:
https://github.com/inverse-inc/fingerbank/blob/master/rhel/fingerbank.spec#L41

**Comment by dwlfrth** [comment]
@fdurand: Can you confirm you adjust the debian packaging ?

**Comment by dwlfrth** [comment]
Fixed by @fdurand with commit 4e08acc7c7d860312f293fe9765b45d79d49f83f
